networking/eth2/src/integration-test/java/tech/pegasys/teku/networking/eth2/BeaconBlocksByRangeIntegrationTest.java

```diff
@@ -204,6 +205,10 @@ public void requestBlockByRange_withDisparateVersionsEnabled_requestNextSpecBloc
       final SpecMilestone nextMilestone,
       final boolean nextSpecEnabledLocally,
       final boolean nextSpecEnabledRemotely) {
+    if (nextMilestone.equals(GLOAS) && nextSpecEnabledRemotely) {
+      // TODO gloas
```

Comment:
We gonna forget to add this if we dont have a ticket :)

Reply:
i've deleted that code, but if i do re-enable it we'll raise tickets...